### PR TITLE
Fixes to paths

### PR DESCRIPTION
**Pull request by donker**
The current implementation will not work on virtual roots and expects DNN to be in the root. This PR attempts to fix those errors.

### CODE REVIEW

File: app/visualizers/view/controller.js [hunk 1]
```diff
@@ -1,5 +1,6 @@
 ﻿app.controller('visualizerController', ['$scope', '$q', '$uibModal', '$sce', '$window', 'toastr', 'visualizerService', function ($scope, $q, $uibModal, $sce, $window, toastr, visualizerService) {
 
+    console.log('visualizerController', siteRoot);
```
Comment (valadas) [inline app/visualizers/view/controller.js]: Can we remove this console log please.

File: manifest.dnn [hunk 2]
```diff
@@ -66,14 +66,6 @@
...
           </scripts>
         </component>
-        <component type="ResourceFile">
-          <resourceFiles>
-            <basePath>bin</basePath>
-            <resourceFile>
-              <name>symbols.zip</name>
-            </resourceFile>
-          </resourceFiles>
-        </component>
```
Comment (valadas) [inline manifest.dnn]: Any reason to remove this ?

**Review by valadas — CHANGES_REQUESTED** [review]
Great catchs, thanks for this massive PR, just 2 small nits...

**Review by valadas — CHANGES_REQUESTED** [review]
Also digging a bit further into this, isn't the purpose of ResolveUrl to give back a URL for a control path, here I think we could just use the server path instead of a URL to RegisterScript no ?

File: manifest.dnn (hunk 2 above)
Comment (donker) [reply manifest.dnn]: Why would that be useful in production?

Comment (mitchelsellers) [reply manifest.dnn]: It doesn’t necessarily harm things. Especially given the current state of this project

Comment (valadas) [reply manifest.dnn]: It provides better error logging and does not affect performance...

File: app/visualizers/view/controller.js (hunk 1 above)
Comment (valadas) [reply app/visualizers/view/controller.js]: Thank you

File: manifest.dnn (hunk 2 above)
Comment (valadas) [reply manifest.dnn]: Thanks for the changes

File: app/visualizers/view/controller.js (hunk 1 above)
Comment (david-poindexter) [reply app/visualizers/view/controller.js]: Looks like this was removed in the latest commit.  I'll go ahead and mark this conversation as resolved.

**Comment by valadas** [comment]
The build issue here is unrelated to the PR, I'll edit the yaml to force windows-2019 and re-fire this build as a quick fix. Later on I'll do some updates so the build supports windows-latest again (or better and build on ubuntu)...

**Comment by david-poindexter** [comment]
@donker why did you close the PR?  It was literally minutes from being merged.  I just don't get it.

**Pull request closed without merging**